torch/_torch_docs.py

```diff
@@ -752,6 +752,9 @@ def parse_kwargs(desc):
         Non-empty tensors provided must have the same shape, except in the
         cat dimension.
     dim (int, optional): the dimension over which the tensors are concatenated
+    pad (bool, optional): add paddings to expand input tensors to same shape
```

Comment:
this needs more explanation of how the padding is done; you can refer to the paragraph split I mention above.